extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-input.js

```diff
@@ -32,19 +32,26 @@ describes.realWin('amp-recaptcha-input', {
     toggleExperiment(win, 'amp-recaptcha-input', true);
   });
 
-  function getRecaptchaInput(skipSiteKey, skipAction) {
+  function getRecaptchaInput(config = {}) {
```

Comment:
I find `createElementWithAttributes` could be helpful here. We could make the input the attributes object. 
The `skipName` `skipSitekey`.. can be a bit confusing.